yarn-project/archiver/package.json

```diff
@@ -6,24 +6,24 @@
   "typedoc": {
     "entryPoint": "./src/index.ts",
     "displayName": "Archiver",
-    "tsconfig": "./tsconfig.dest.json"
+    "tsconfig": "./tsconfig.json"
   },
   "scripts": {
     "prepare": "node ../yarn-project-base/scripts/update_build_manifest.mjs package.json",
     "prepare:check": "node ../yarn-project-base/scripts/update_build_manifest.mjs package.json --check",
-    "build": "yarn clean && tsc -b tsconfig.dest.json",
-    "build:dev": "tsc -b tsconfig.dest.json --watch",
+    "build": "yarn clean && tsc -b tsconfig.json",
```

Comment:
We can do this in a followup PR but because tsconfig.json is the default name, we can actually drop all these -b and p flags

Reply:
Or I can change it really quick with a yarn prepare!